vavr/src/test/java/io/vavr/collection/StreamTest.java

```diff
@@ -198,11 +203,73 @@
         return Stream.transpose((Stream<Stream<T>>) rows);
     }
 
+    //fixme: delete, when useIsEqualToInsteadOfIsSameAs() will be eliminated from AbstractValueTest class
     @Override
     protected boolean useIsEqualToInsteadOfIsSameAs() {
         return true;
     }
 
+    @Test
+    @Override
+    public void shouldRemoveNonExistingElement() {
+        final Seq<Integer> t = of(1, 2, 3);
+        assertThat(t.remove(4)).isEqualTo(t);
+        assertNotSame(t, t.remove(4));
```

Comment:
All current test is used in form `assertThat(actual).testMethod(expected);` and more - `assertThat(actual)` is support chaining: `assertThat(t.remove(4)).isEqualTo(t).isNotSameAs(t);`

Reply:
I didn't khow that! Thanks! I'll change my code